playbooks/tasks/portal-role-task-certbot-custom-dns.yml

```diff
@@ -0,0 +1,10 @@
+---
+# Check for custom DNS and setup accordingly
+#
+# NOTE: currently the only custom setup that is handled is Cloudflare. Logic in
+# this file should be updated to only allow 1 custom config when others are
+# added.
+
+- name: Setup Cloudflare DNS for Certbot
+  include_tasks: tasks/portal-role-task-certbot-setup-cloudflare.yml
+  when: dns_cloudflare_api_token is defined
```

Comment:
Where is the secret value fo `dns_cloudflare_api_token` defined?
In hosts.ini or cluster yamls?
- this is not secured
=> it should be defined in secrets storage (cluster config) I think.